docs/developer-guide/plugins.md

```diff
@@ -87,52 +86,55 @@ returns an observable of the response. It can be used to modify the request and/
 
 #### The intercept() method
 
-The intercept() method is called when an HTTP request is available for plugin to process. Note that, however:
+The `intercept` is a callback that exposes a proxied request to a HttpInterceptor (plugin). 
+Note that:
 
-* intercept() methods are called serially, in order they are configured in the pipeline, 
+* `intercept` methods are called serially, in order they are configured in the pipeline, 
   starting from the first interceptor.
-* an interceptor "owns" the request until it is ready to pass it on to the next one. 
-  It is at this point the intercept() of the next interceptor is called.
-* A plugin may choose to respond to the request, in which case the request will not 
+* an interceptor "owns" the request until it passes it to the next interceptor, by
+  calling `chain.proceed`. At this point Styx calls `intercept` of the next interceptor.
+* If a plugin chooses to respond to the request, the request will not 
   get propagated any further down the pipeline.
 
-The intercept method is called with two arguments, a HttpRequest instance (self-explanatory), 
-and Chain. The Chain object represents a view of the Styx interceptor pipeline as for this 
-particular request. The `intercept()` method will use the Chain object, most importantly, to 
-pass the request object onwards in the pipeline. It can also be used to store some 
-request-specific context that is visible for other plugins.
+The `intercept` is called with two arguments: HTTP request, and a `Chain`. 
+The `Chain` represents a Styx interceptor pipeline as for this request, and
```

Comment:
Perhaps we could say 

"The `Chain` represents the section of the Styx interceptor pipeline further downstream from this plugin"?

Or something like that.

Reply:
ok